docs/docs/getting-started/installation.mdx

```diff
@@ -18,9 +18,9 @@ All you need to do, to install Mapperly is to add a NuGet reference pointing to
 <!-- do not indent this, it won't work, https://stackoverflow.com/a/67579641/3302887 -->
 
 <Tabs>
-<TabItem value="csproj" label="PackageReference" default>
-    <CodeBlock language="xml">{`<PackageReference Include="Riok.Mapperly" Version="${useDocusaurusContext().siteConfig.customFields.mapperlyVersion}" />`}</CodeBlock>
-</TabItem>
+  <TabItem value="csproj" label="PackageReference" default>
```

Comment:
Do not ident this or remove comment above.

Reply:
Fixed